src/App.tsx

```diff
@@ -1,14 +1,48 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoUser } from './types/User';
 
 export const App: React.FC = () => {
+  const [loaded, setLoaded] = useState(false);
+  const [sourceData, setSourceData] = useState<Todo[]>([]);
+  const [sortBy, setSortBy] = useState('all');
+  const [userModal, setUserModal] = useState<TodoUser | null>(null);
+  const [showModal, setShowModal] = useState(false);
+  const [eyeMark, setEyeMark] = useState<number>(-1);
+  const sortTodos = () => {
+    if (loaded) {
+      switch (sortBy) {
+        case 'active':
```

Comment:
Move all cases to enum

Reply:
its no make sense, human has no access to select values and never use them in code